mne/viz/_3d.py

```diff
@@ -1892,7 +1893,11 @@ def _plot_stc(stc, subject, surface, hemi, colormap, time_label,
             kwargs["fmax"] = scale_pts[2]
             kwargs["clim"] = clim
             kwargs["volume_options"] = volume_options
-            kwargs["src"] = src_vol
+            if src_vol is not None:
+                kwargs["src"] = src_vol
+            else:
+                kwargs["src"] = orig_src
```

Comment:
I suspect this is a bug, for example if it's a mixed source estimate (surface+vol). I think now that we use `src` for both surface and vol, it just juld be `kwargs['src'] = orig_src`, and the logic for extracting the volume parts should be moved to `Brain` init (and really it should become logic for splitting into volume and surface parts).